src/App.jsx

```diff
@@ -1,14 +1,10 @@
 import React from 'react';
 import './App.css';
-import PageScan from './components/PageScan';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <PageScan />
-      </header>
-    </div>
+    <>
+    </>
```

Comment:
this makes the App empty...

Reply:
yes, that's normal

Reply:
if you say so... still the name ScanePage is strange